src/style.css

```diff
@@ -2,2 +1,106 @@
+.roboto-regular {
+  font-family: Roboto, serif;
+  font-style: normal;
+}
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
+.container {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  background: #fff;
+  padding: 0 20px;
+}
+
+.moyoLogo {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 40px;
+  height: 40px;
+  margin-left: 30px;
+}
+
+.nav-links {
+  transform: translateX(-30px);
+  height: 60px;
+  margin-left: auto;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.nav-links ul {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  list-style: none;
+}
+
+.nav-links li + li {
+  margin-left: 20px;
+  line-height: 60px;
+}
+
+.nav-item {
+  font-size: 12px;
+  text-transform: uppercase;
+  color: #000;
+  text-decoration: none;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+}
+
+.nav-item:hover {
+  color: rgba(0, 172, 220, 1);
+}
+
+.is-active {
+  display: flex;
+  color: rgba(0, 172, 220, 1);
+  align-items: center;
+  justify-content: center;
+  background: white;
+  line-height: 60px;
+}
+
+a.is-active {
+  display: flex;
+  align-items: center;
+  background-color: #fff;
+}
+
+.is-active::after {
+  content: '   ';
+  width: 100%;
+  height: 4px;
+  background-color: rgba(0, 172, 220, 1);
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  border-radius: 8px;
+}
+
+@media (max-width: 1200px) {
+  html {
+    width: 100%;
+  }
+}
+
+@media (max-width: 1024px) {
+  html {
+    width: 100%;
+  }
```

Comment:
The media queries for max-width 1200px and 1024px both set the `html` width to 100%. If there are no additional styles specific to these breakpoints, you might consider combining them or ensuring they serve distinct purposes.